accounts-db/src/accounts.rs

```diff
@@ -44,8 +44,25 @@ use {
     },
 };
 
+// max number of accts an entry can hold: 64 * 256
+const ENTRY_ACCTS_LOOKUP_TABLE_SIZE: usize = 16384;
```

Comment:
this could be a reasonable default size, but there's no strict limit on number of txs in an entry.

Reply:
also this isn't a hard cap as implied by the use of the word `max` in the comment above...